api/rest/client/methods.go

```diff
@@ -172,3 +173,15 @@ func (c *Client) GetConnectGraph() (api.ConnectGraphSerial, error) {
 	err := c.do("GET", "/health/graph", nil, &graphS)
 	return graphS, err
 }
+
+// AddMultiFile adds new files to the ipfs cluster, importing and potentially
+// sharding underlying dags across the ipfs repos of multiple cluster peers
+func (c *Client) AddMultiFile(multiFileR *files.MultiFileReader) error {
```

Comment:
It's good to have a method to send an already-prepared multifile thing, but it would expect also an `Add(io.Reader)` method that does a multifile upload. Don't know if now or later, but just to keep in mind.